### PR TITLE
Remove allowlist entries for `_ctypes` on win

### PR DESCRIPTION
**Pull request by sobolevn**
Source: https://github.com/python/cpython/blob/64f158e7b09e67d0bf5c8603ff88c86ed4e8f8fd/Modules/_ctypes/callproc.c#L1361-L1433

### CODE REVIEW

File: stdlib/_ctypes.pyi [hunk 1]
```diff
@@ -44,6 +44,8 @@ if sys.platform == "win32":
     def FormatError(code: int = ...) -> str: ...
     def get_last_error() -> int: ...
     def set_last_error(value: int) -> int: ...
+    def LoadLibrary(__name: str, __load_flags: int) -> int: ...
```
Comment (srittau) [inline stdlib/_ctypes.pyi:47]:
```suggestion
    def LoadLibrary(__name: str, __load_flags: int = 0) -> int: ...
```

**Review by srittau — CHANGES_REQUESTED** [review]
Thanks! I believe `load_flags` has a default value, though:

https://github.com/python/cpython/blob/64f158e7b09e67d0bf5c8603ff88c86ed4e8f8fd/Modules/_ctypes/callproc.c#L1375C10-L1375C26

File: stdlib/_ctypes.pyi (hunk 1 above)
Comment (AlexWaygood) [reply stdlib/_ctypes.pyi:47]: Can confirm from local testing on a Windows machine.

Comment (sobolevn) [reply stdlib/_ctypes.pyi:47]: Yes, it does have `int load_flags = 0;`

**Review by AlexWaygood — APPROVED** [review]
lgtm

**Review by srittau — APPROVED** [review]

**Pull request merged**